README.md

```diff
@@ -219,6 +333,22 @@ When using politeiawww_refclient, the `-use-paywall` flag is true by default. Wh
  * Set the user created in the first refclient execution as admin with politeiawww_dbutil.
  * Run refclient again with the `email` and `password` flags set to the user created in the first refclient execution.
 
+#### Cache
+
+##### Rebuilding the Cache
+
+The cache can be rebuilt using the `--buildcache` flag when starting
+`politeiad`.  This will drop all current tables from the politeia database,
+re-create the tables, then populate the new tables with the data that is in the
+politeiad git repositories.
+
+##### Testnet vs. Mainnet Cache
+
+If you are switching between running politeia on testnet and mainnet, you will
```

Comment:
This must be automatic. We can key off the database name and use chainparams for the name which matches what we currently do when populating the data dir.

Reply:
Fixed.

Note: the cockroachdb data directory is not set in politeiad, it is set when you start cockroachdb.